.changeset/heavy-lizards-bow.md

```diff
@@ -0,0 +1,5 @@
+---
+"@utrecht/link-social-css": patch
```

Comment:
Omdat de `@utrecht/design-tokens` package is gewijzigd, zou je deze willen gebruiken in plaats van `@utrecht/link-social-css`?
Zou je ook nog kunnen beschrijven wat er precies is gewijzigd? Bijvoorbeeld:


```md
Changed the background-color of the `link-social` component from `utrecht.color.red.40` to `utrecht.color.blue.35`  

```